Algebraic Topology/algebraic-topology.tex

```diff
@@ -156,7 +156,7 @@ \section{Homotopy}
 \mydate{d2}{20}{1}{2017}
 
 \begin{defn}
-  A \de{category} is a directed graph.  Each vertex is called an
+  A \de{category} is a directed graph. Each vertex is called an %
```

Comment:
Either solution is fine with me.  I would love to know the correct definition of a category!